packages/webapp/src/Routes.jsx

```diff
@@ -544,7 +550,8 @@ const Routes = () => {
             <Route path="/fence/:location_id" component={FenceDetails} />
             <Route path="/buffer_zone/:location_id" component={BufferZoneDetails} />
             <Route path="/watercourse/:location_id" component={WatercourseDetails} />
-
+            <Route path="/sensor/:location_id/details" exact component={SensorDetail} />
+            <Route path="/sensor/:location_id/edit" exact component={EditSensor} />
```

Comment:
FYI, the pattern seems to be to have these routes in a dedicated file under `packages/webapp/src/routes`. For consistency, we should probably move them there